third_party/amd/lib/TritonAMDGPUToLLVM/TargetInfo.cpp

```diff
@@ -0,0 +1,17 @@
+#include "TargetInfo.h"
+#include "amd/include/TritonAMDGPUToLLVM/GCNAsmFormat.h"
+#include "triton/Conversion/TritonGPUToLLVM/Utility.h"
+
+namespace AMD {
+bool TargetInfo::isHwNanPropagationSupported() const { return false; }
+Value TargetInfo::callBallotOp(ConversionPatternRewriter &rewriter,
+                               Location loc, Type type, Value threadMask,
```

Comment:
Maybe remove threadMask argument and hide it inside the api ? Seems its also hardcoded for nvidia backend to `i32_val(-1)`

Reply:
+1